hs/src/Reach/Backend/JS.hs

```diff
@@ -286,7 +286,7 @@ jsPrimApply = \case
   PGE t -> jsApply_ui t "stdlib.ge"
   PGT t -> jsApply_ui t "stdlib.gt"
   SQRT t -> jsApply_ui t "stdlib.sqrt"
-  UCAST x y -> \a -> jsApply "stdlib.cast" $ [ jsUIntTy x, jsUIntTy y ] <> a
+  UCAST dom rng trunc -> \a -> jsApply "stdlib.cast" $ [ jsUIntTy dom, jsUIntTy rng ] <> a <> [ jsBool trunc ]
```

Comment:
Make sure you increment the version number every time you expect something different from the stdlib

Reply:
Which version number?

Reply:
`reachBackendVersion`

Reply:
The JS code will need to be updated to match it